From d695f60666453d964a653c5033764c4b80121f56 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <cera@cera.cz>
Date: Mon, 20 Feb 2017 15:13:52 +0100
Subject: [PATCH 2/3] ethtool: sync help output for -x/-X with man page

Add missing words to the help output for -x and -X commands as well
as an ability to set the indirection table to default value.

Signed-off-by: Ivan Vecera <cera@cera.cz>
---
 ethtool.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/ethtool.c b/ethtool.c
index 75299c6..0d3244e 100644
--- a/ethtool.c
+++ b/ethtool.c
@@ -4645,10 +4645,10 @@ static const struct option {
 	{ "-T|--show-time-stamping", 1, do_tsinfo,
 	  "Show time stamping capabilities" },
 	{ "-x|--show-rxfh-indir|--show-rxfh", 1, do_grxfh,
-	  "Show Rx flow hash indirection and/or hash key" },
+	  "Show Rx flow hash indirection table and/or RSS hash key" },
 	{ "-X|--set-rxfh-indir|--rxfh", 1, do_srxfh,
-	  "Set Rx flow hash indirection and/or hash key",
-	  "		[ equal N | weight W0 W1 ... ]\n"
+	  "Set Rx flow hash indirection table and/or RSS hash key",
+	  "		[ equal N | weight W0 W1 ... | default ]\n"
 	  "		[ hkey %x:%x:%x:%x:%x:.... ]\n" },
 	{ "-f|--flash", 1, do_flash,
 	  "Flash firmware image from the specified file to a region on the device",
-- 
1.8.3.1

